providers/azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/PublicIPAddressApiLiveTest.java

```diff
@@ -119,6 +119,16 @@ public void listPublicIPAddresses() {
       assertTrue(ipList.size() > 0);
    }
 
+   @Test(groups = "live", dependsOnMethods = "createPublicIPAddress")
+   public void listAllPublicIPAddresses() {
```

Comment:
Same here. Add it as a dependency to the delete test.

Reply:
same :)